WordPress/src/main/res/values/strings.xml

```diff
@@ -3344,7 +3344,8 @@
     <string name="new_site_creation_site_name_header_subtitle">A good name is short and memorable.\nYou can change it later.</string>
 
     <!-- Site Creation : Domains -->
-    <string name="site_creation_domain_cost">%s/yr</string>
+    <string name="site_creation_domain_cost">per year</string>
```

Comment:
In theory, it'd be great if we can address the issue at the source. However, I am not sure if we can do it in practice. The main issue is that, even though this string is changed in `trunk` and ready to be reset in GlotPress - it's still in use in the current code freeze branch, so the prior translations are still necessary. I might be missing a possibility, but as far as I know our recommendation is to change the key in these cases.

Just in case I am missing something, I'll cc @AliSoftware (who I think is the most knowledgeable in localization) to weigh in as well.

Reply:
>  I might be missing a possibility, but as far as I know our recommendation is to change the key in these cases.

That is 💯  correct, for the exact reasons Oguz mentioned